packages/ptokens-assets-evm/src/ptokens-evm-provider.ts

```diff
@@ -36,41 +27,46 @@ export type MakeContractCallOptions = {
   /** The method to be called. */
   method: string
   /** The contract ABI. */
-  abi: ContractAbi
+  abi: Abi
   /** The contract address. */
   contractAddress: string
 }
 
-class SendOptions implements PayableTxOptions {
-  from: string
-  value: string
-  gasPrice: string
-  gas: string
-  constructor(from: string, value: string) {
-    this.from = from
-    this.value = value
-  }
-  maybeSetGasPrice(gasPrice: number) {
-    if (gasPrice) this.gasPrice = gasPrice.toString()
-    return this
-  }
-  maybeSetGasLimit(gasLimit: number) {
-    if (gasLimit) this.gas = gasLimit.toString()
-    return this
-  }
-}
+// class SendObject<T> {
```

Comment:
Can't you re-enable this? There should be a way to set gas price/limit in viem as well.